packages/esm-ward-app/src/types/index.ts

```diff
@@ -76,6 +76,17 @@ interface BedTagMap {
 
 export type BedStatus = 'AVAILABLE' | 'OCCUPIED';
 
+export type DispositionType = 'ADMISSION' | 'TRANSFER' | 'DISCHARGE';
+
+export interface InpatientRequest {
+  patient: Patient;
+  visit: Visit;
+  type: DispositionType;
+  encounter?: Encounter;
+  dispositionObs?: Observation;
+  dispositionLocation?: Location;
+  dispositionDate?: Date;
```

Comment:
FWIW, if we use the current PR, this the request will just include patient and visit for now, though we will have to add the "type" sooner rather than later.